diff --git a/dist/preset.js b/dist/preset.cjs
similarity index 100%
rename from dist/preset.js
rename to dist/preset.cjs
diff --git a/package.json b/package.json
index 0288916206b88d81f4e40d8278261ebbbc2c9ff8..8015755797aef8891746fa1d8fa7fee90054c428 100644
--- a/package.json
+++ b/package.json
@@ -29,8 +29,9 @@
     },
     "./preset": {
       "types": "./dist/preset.d.ts",
-      "require": "./dist/preset.js",
-      "import": "./dist/preset.mjs"
+      "require": "./dist/preset.cjs",
+      "import": "./dist/preset.cjs",
+      "default": "./dist/preset.cjs"
     },
     "./package.json": "./package.json"
   },
diff --git a/preset.js b/preset.js
index 20cd9ba34616097e988a07adb88505f03a21a535..1af27e653b97acdc737f4353f1739fdef4f52156 100644
--- a/preset.js
+++ b/preset.js
@@ -1,2 +1,6 @@
-// eslint-disable-next-line @typescript-eslint/no-require-imports
-module.exports = require('./dist/preset');
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+const preset = require("./dist/preset.cjs");
+export const core = preset.core;
+export const viteFinal = preset.viteFinal;
+export default preset;
